xbmc/cores/dvdplayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -45,6 +45,8 @@ CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IDVDPlayer* pPlayer,
 {
   CFileItem item(file.c_str(), false);
 
+  item.SetMimeType(content);
```

Comment:
I can't see/find a place where the mime type is used before line 108. Why is this call needed? Maybe I've overlooked something?